modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -386,57 +398,171 @@ Examples: `10.10.10.0/24` or `100:100:100:100::/64`
+| `direction` optionally defines a direction to filter flows by. The available options are `Ingress` and `Egress`.
 
 | `enable`
 | `boolean`
 | Set `enable` to `true` to enable the eBPF flow filtering feature.
 
 | `icmpCode`
 | `integer`
-| `icmpCode`, for Internet Control Message Protocol (ICMP) traffic, defines the ICMP code to filter flows by.
+| `icmpCode`, for Internet Control Message Protocol (ICMP) traffic, optionally defines the ICMP code to filter flows by.
 
 | `icmpType`
 | `integer`
-| `icmpType`, for ICMP traffic, defines the ICMP type to filter flows by.
+| `icmpType`, for ICMP traffic, optionally defines the ICMP type to filter flows by.
+
+| `peerCIDR`
+| `string`
+| `peerCIDR` defines the Peer IP CIDR to filter flows by.
+Examples: `10.10.10.0/24` or `100:100:100:100::/64`
 
 | `peerIP`
 | `string`
-| `peerIP` defines the IP address to filter flows by.
+| `peerIP` optionally defines the remote IP address to filter flows by.
 Example: `10.10.10.10`.
 
 | `pktDrops`
 | `boolean`
-| `pktDrops` filters flows with packet drops
+| `pktDrops` optionally filters only flows containing packet drops.
 
 | `ports`
 | `integer-or-string`
-| `ports` defines the ports to filter flows by. It is used both for source and destination ports.
+| `ports` optionally defines the ports to filter flows by. It is used both for source and destination ports.
 To filter a single port, set a single port as an integer value. For example, `ports: 80`.
 To filter a range of ports, use a "start-end" range in string format. For example, `ports: "80-100"`.
 To filter two ports, use a "port1,port2" in string format. For example, `ports: "80,100"`.
 
 | `protocol`
 | `string`
-| `protocol` defines the protocol to filter flows by.
+| `protocol` optionally defines a protocol to filter flows by. The available options are `TCP`, `UDP`, `ICMP`, `ICMPv6`, and `SCTP`.
+
+| `rules`
+| `array`
+| `flowFilterRules` defines a list of ebpf agent flow filtering rules
+
+| `sampling`
+| `integer`
+| `sampling` sampling rate for the matched flow
 
 | `sourcePorts`
 | `integer-or-string`
-| `sourcePorts` defines the source ports to filter flows by.
+| `sourcePorts` optionally defines the source ports to filter flows by.
 To filter a single port, set a single port as an integer value. For example, `sourcePorts: 80`.
 To filter a range of ports, use a "start-end" range in string format. For example, `sourcePorts: "80-100"`.
 To filter two ports, use a "port1,port2" in string format. For example, `ports: "80,100"`.
 
 | `tcpFlags`
 | `string`
-| `tcpFlags` defines the TCP flags to filter flows by.
+| `tcpFlags` optionally defines TCP flags to filter flows by.
+In addition to the standard flags (RFC-9293), you can also filter by one of the three following combinations: `SYN-ACK`, `FIN-ACK`, and `RST-ACK`.
+
+|===
+== .spec.agent.ebpf.flowFilter.rules
+Description::
++
+--
+`flowFilterRules` defines a list of ebpf agent flow filtering rules
```

Comment:
need to mention its dev preview here or above or both. Dont know the norm

Reply:
See earlier Dev Preview concern.